src/routes/(app)/admin[[hash=admin_hash]]/product/[id]/subscribers/+page.server.ts

```diff
@@ -16,6 +16,10 @@ export const load = async ({ params }) => {
 		product: pojo(product),
 		subscriptions: subscriptions.map((subscription) => ({
 			...subscription,
```

Comment:
```suggestion
			...pojo(subscription),
```

this would have worked too